joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/astgen/AstGenRunner.scala

```diff
@@ -181,9 +181,13 @@ trait AstGenRunnerBase(config: X2CpgConfig[?] & AstGenConfig[?]) {
         val parsed  = filterFiles(srcFiles, out)
         val skipped = skippedFiles(in, result.toList)
         DefaultAstGenRunnerResult(parsed, skipped)
+      case Failure(f) if !isRetry =>
```

Comment:
I do not think is a good idea. Everytime some astgen failing this will retry the invocation. In case of very large project scans failing at the very end (maybe even failing for a good reason) this may lead to very long runtimes.

Reply:
Ok will remove